### PR TITLE
Event sign in form (WIP)

### PR DESCRIPTION
**Pull request by thaigillespie**
Still WIP

### CODE REVIEW

**Comment by godwinpang** [comment]
lgtm - honestly feel free to merge in if you want to since I think 99% of the UI is there already. We'll just need to add the confirmation alert and hook it up with the backend. Great work @thaigillespie !

File: package.json [hunk 1]
```diff
@@ -28,6 +28,7 @@
     "prop-types": "^15.7.2",
     "react": "^16.13.1",
     "react-app-rewired": "^2.1.6",
+    "react-date-range": "^1.0.3",
```
Comment (godwinpang) [inline package.json:31]: jw - why is this added?

Comment (thaigillespie) [reply package.json:31]: Err I messed up on the dependencies from dateFilter lol : ((

**Comment by thaigillespie** [comment]
I'll merge it in for now, I'll make a separate PR for the confirmation modal
